getting-started/forge-multi-node.md

```diff
@@ -0,0 +1,256 @@
+# Setting up TimescaleDB 2.0 Multi-node on Timescale Forge
+
+TimescaleDB 2.0  [introduces a number of new features][changes-in-tsdb2] 
+to supercharge time-series data even further. One of the most anticipated new features 
+is what we call **multi-node** - the ability to create a cluster of TimescaleDB 
+instances to scale both reads and writes. 
+
+In this how-to, we’ll show you how to create a multi-node cluster in your Timescale 
+Forge account using TimescaleDB 2.0-RC3. 
+
+This document details a “DIY” multi-node experience on Forge until the console UI 
+provides thetools to do it. In the future we will offer a point-and-click multi-node 
+experience within the Timescale Forge console UI, but we wanted to provide a way 
+for users to setup and test this feature before that interface is available. 
+
+If you’re ready to set up a TimescaleDB multi-node cluster as part of your Timescale 
+Forge account, continue on! If you want to follow along but don't yet have a Timescale 
+Forge account, [sign-up today for a 100% free, 30-day trial][sign-up]!
+
+## Overview of multi-node setup
+
+Multi-node clusters consist of at least two or more TimescaleDB instances 
+(called **Services** in Timescale Forge). Each cluster has one access node (AN) 
+and one or more data nodes (DN). As outlined in our [architecture blog posts][distributed-architechture], 
+the access node is intended to be the only TimescaleDB instance that you or your 
+applications connect to once the cluster is set up. It becomes the “brains” and 
+traffic controller of all distributed hypertable activity. In contrast, data nodes 
+are not intended to be accessed directly once joined to a multi-node cluster.
+
+>:TIP:A proper TimescaleDB cluster should have at least two data nodes to begin 
+realizing the benefits of distributed hypertables. While it is technically possible
+to add just one data node to a cluster, this will often perform worse than a 
+single TimescaleDB instance and is not recommended. 
+
+## Step 1: Create Services for Access and Data node Services
+
+First, you need to create new Services within your Forge account. As mentioned 
+earlier, you should create _at least_ three Services to set up a multi-node cluster: 
+one access node and two data nodes. 
+
+Because there is currently no way to distinguish between the access node and data 
+nodes within the Timescale Forge console, **we strongly recommend that you include 
+“AN” and “DN” in the names of each service, respectively (eg. “AN-mycluster”, 
```

Comment:
imho more natural naming is `mycluster-an` or `mycluster-dn`

Reply:
"... there are two hard things in programming..." 😂. I don't feel strongly either way, the only advantage of this for the how-to is that it's easier for me to spot up front in the SQL when 'dn1' is using 'DN1-mycluster'.